src/Features/Core/Portable/SpellCheck/AbstractSpellCheckCodeFixProvider.cs

```diff
@@ -37,7 +37,7 @@ internal abstract class AbstractSpellCheckCodeFixProvider<TSimpleName> : CodeFix
                 // Only bother with identifiers that are at least 3 characters long.
                 // We don't want to be too noisy as you're just starting to type something.
                 var nameText = name.GetFirstToken().ValueText;
-                if (nameText.Length >= 3)
+                if (nameText?.Length >= 3)
```

Comment:
See #6338 Any time the name doesn't exist so in `Microsoft.CodeAnalysis.` the name after the last dot does not exist and is null